infrastructures/openshift/pom.xml

```diff
@@ -55,6 +55,10 @@
             <groupId>javax.inject</groupId>
             <artifactId>javax.inject</artifactId>
         </dependency>
+        <dependency>
+            <groupId>javax.ws.rs</groupId>
+            <artifactId>javax.ws.rs-api</artifactId>
```

Comment:
is this dependency needs to be declared?

Reply:
Maven dependency plugin asked me to add it on some development phase. Not it is unused and I've removed it.